dae-alright/src/components/Button/Button.component.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Button.style.scss';
+
+const Button = ({btnDetails,bgColor, color, icons, width, height, className}) => (
```

Comment:
I think four props are needed here, `text`, `style`, and `className`

Reply:
Done.